tests/Doctrine/Tests/DBAL/Functional/Schema/MySqlSchemaManagerTest.php

```diff
@@ -454,21 +454,7 @@ public function testEnsureDefaultsAreUnescapedFromSchemaIntrospection() : void
         $platform = $this->_sm->getDatabasePlatform();
```

Comment:
This entire test should go to `SchemaManagerFunctionalTestCase`.

Reply:
I didn't pay enough attention to it. I now realize that it is a duplicate of `SchemaManagerFunctionalTestCase::testEscapedDefaultValueMustBePreserved()`. I'll remove this one.